rotkehlchen/chain/evm/decoding/socket_bridge/constants.py

```diff
@@ -0,0 +1,7 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+CPT_SOCKET: Final = 'socket'
+GATEWAY_ADDRESS: Final = string_to_evm_address('0x3a23F943181408EAC424116Af7b7790c94Cb97a5')
```

Comment:
Is it the same in every EVM chain? That's weird.

Reply:
Yes, it is stated here https://docs.socket.tech/socket-api/contract-addresses